serverless/serverless-support.adoc

```diff
@@ -6,20 +6,27 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-include::modules/serverless-getting-support.adoc[leveloffset=+1]
+If you experience difficulty with a procedure described in this documentation, visit the Red Hat Customer Portal at http://access.redhat.com. You can use the Red Hat Customer Portal to search or browse through the Red Hat Knowledgebase of technical support articles about Red Hat products. You can also submit a support case to Red Hat Global Support Services (GSS), or access other product documentation.
+
+If you have a suggestion for improving this guide or have found an error, please submit a Bugzilla report at http://bugzilla.redhat.com against *Product* for the *Documentation* component. Provide specific details, such as the section number, guide name, and {ServerlessProductName} version so we can easily locate the content.
```

Comment:
ISG says to avoid using the word please, except for marketing content. Suggested edit:
```suggestion
If you have a suggestion for improving this guide or have found an error, you can submit a Bugzilla report at http://bugzilla.redhat.com against *Product* for the *Documentation* component. Provide specific details, such as the section number, guide name, and {ServerlessProductName} version so we can easily locate the content.
```